components/admin_console/database_settings.jsx

```diff
@@ -25,7 +25,7 @@ export default class DatabaseSettings extends AdminSettings {
 
     getConfigFromState(config) {
         // driverName and dataSource are read-only from the UI
-
+        config.SqlSettings.ConnMaxLifetimeMilliseconds = this.parseIntNonNegative(this.state.connMaxLifetimeMilliseconds);
```

Comment:
I could have also used this.parseIntNonZero(this.state.connMaxLifetimeMillseconds, 0) but I didn't think it was logical to pass in 0 as the default value to a function called parseIntNonZero. Let me know what you prefer!

Reply:
Good call: I like what you've done :)